src/components/Button/ButtonContent/index.test.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { ButtonContent, layoutType } from '.'
+import renderWithTheme from '../../../testing/renderWithTheme'
+import mentor from '../../../assets/mentor.svg'
+
+test('renders button content with layout center', () => {
+	const { baseElement } = renderWithTheme(
+		<ButtonContent layout={layoutType.center} />
+	)
+
+	expect(baseElement).toMatchInlineSnapshot(`
+		<body>
+		  <div>
+		    <div
+		      class="sc-hKgILt eNcJhZ"
```

Comment:
Co je to za classes? Nevím, jestli nepatří k nějaké externí knihovně nebo k těm svgčkům, ale jestli jsou naše, tak bychom je možná měly pojmenovávat nějak za jedno dle naší konvence maleVelkePismeno a aby bylo poznat, co dělají. :thinking:

Reply:
Ne, pokud to chápu dobře, tak tyhle třídy generuje React. My to máme vše dělané ve styled component žádné ccs s class. Je to asi stejné jako když v Reactu píšeš Link, ale v Browseru se ti objeví tag a.

Reply:
Info: Classy generuje Styled Components a je to bráno jako výhoda, že nikde nedojde k duplikaci, překrývání nebo překlepům.